classes/article/ArticleDAO.inc.php

```diff
@@ -182,11 +183,14 @@ function updateObject($article) {
 		);
 
 		$this->updateLocaleFields($article);
+		$contextId = Request::getContext()->getId();
```

Comment:
I think you do not need that parameter at all -- if you see my other comments... Also, Alec proposed to have another DB column in the submissions table with the current/latest version, I believe.... So maybe to first go roughly over all comments and then start with changes... ?